torch/functional.py

```diff
@@ -981,6 +984,104 @@ def cdist(x1, x2, p=2., compute_mode='use_mm_for_euclid_dist_if_necessary'):
     else:
         raise ValueError("{} is not a valid value for compute_mode".format(compute_mode))
 
+def atleast_1d(*tensors):
+    r"""
+    atleast_1d(input) -> Tensor
+    Returns a 1-dimensional view of each each input tensor with zero dimensions.
+    Input tensors with one or more dimensions are returned as-is.
+
+    Args:
+        input (Tensor or list of Tensors)
+    Example::
+        >>> x = torch.randn(2)
+        >>> x
+        tensor([1.4584, 0.7583])
+        >>> torch.atleast_1d(x)
+        tensor([1.4584, 0.7583])
+        >>> x = torch.tensor(1.)
+        >>> x
+        tensor(1.)
+        >>> torch.atleast_1d(x)
+        tensor([1.])
+        >>> x = torch.tensor(0.5)
+        >>> y = torch.tensor(1.)
+        >>> torch.atleast_1d((x,y))
+        (tensor([0.5000]), tensor([1.]))
```

Comment:
List or tuple of tensors? NumPy returns a list.

Reply:
I guess this is how Pytorch returns sequence of tensors. I have also seen that for `meshgrid` and `broadcast_tensors`
```python
>>> x = torch.randn(3)
>>> type(torch.meshgrid(x,x,x))
<class 'tuple'>
>>> type(torch.broadcast_tensors(x,x,x))
<class 'tuple'>

```

Reply:
OK. I guess the docs should just say "Tensor or tuple of Tensors", then?

Reply:
Makes sense. Will do that thanks!